### DISCUSSION TITLE
How to close account?

### DISCUSSION THREAD

**Discussion by bafomet88** [swellstores/community#537] [Help] [answered]
Finally, I managed to develop my own solution, so I would like to say goodbay to unstable swell. But I encountered typical for this platform problem - I could no find the way to close account! In the help center there is an information that the button to close account is placed in account tab:
https://www.swell.is/help/store-setup/account

But there is no button in mine dashboard. Looking for help, swell has already charged me for unused account :(

**Comment by logeshswell** [comment]
Can you email support@swell.is regarding the closure of the Swell account with the store details from your registered email?

**Comment by bafomet88** [comment]
If this is the only way, then You should correct documentation, where it is stated, that I can find a button to suspend account any time :/

**Reply by steven-k1** [reply, marked as answer]
@bafomet88 It's possible to do so on your end without contacting support.

To cancel your account, you need to navigate to the account tab within your account's dashboard.
​
Once in the account tab, select the 'Change plan' option. (screenshot below)

![image](https://github.com/swellstores/community/assets/77881204/29d782d9-7077-4246-b586-43b5e8fbca84)

Once within the 'Plans' tab, at the bottom of the page is the 'Cancel this account' option. (screenshot below)

![image](https://github.com/swellstores/community/assets/77881204/a1074067-4099-4fdd-8990-73d509dfbf5f)
